doc/server/play29.md

```diff
@@ -0,0 +1,117 @@
+# Running as a Play 2.9 server
+
+To expose endpoint as a [play-server](https://www.playframework.com/) first add the following dependencies:
+
+```scala
+"com.softwaremill.sttp.tapir" %% "tapir-play-server" % "@VERSION@"
```

Comment:
I think this would need different artifact names (with `29`)?
But maybe we could have a single documentation page for JSON, I think the only difference would be the name of the artifact - so we could simply say that if you are on 3.0, use this artifact, if you are on 2.9, use this other one